content/kapp-controller/docs/latest/packaging-workflows.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Packaging Workflows
+---
+
+We recommend reading through the docs about the new [packaging
+apis](packaging.md) to familiarize yourself with the yaml used in these
+workflows.
+
+## Package Authoring
+
+## Package Consumption
+
+This workflow walks through the example contained in
+the [`packaging-demo`](https://github.com/vmware-tanzu/carvel-kapp-controller/tree/dev-packaging/examples/packaging-demo)
+directory and assumes all commands are run from within the directory.
+
+### Setup
+
+First, some [setup of rbac](security-model.md) is necessary and can be done using:
+
+```
+$ kapp deploy -a default-ns-rbac -f https://raw.githubusercontent.com/vmware-tanzu/carvel-kapp-controller/develop/examples/rbac/default-ns.yml
+```
+
+### Adding the Repo
```

Comment:
I think it might be a good idea to add a section for adding a package to a cluster directly. Use case: As a package author, I might want to test if my package works correctly. For that, I do not want to always bundle it in a repository.

Reply:
Maybe we can place that in the package authors section as a step before including the package in your repo

Reply:
Yeah, that would be good.